dlp/snippets/inspect/inspect_test.go

```diff
@@ -277,12 +277,28 @@ func TestInspectBigquery(t *testing.T) {
 	}
 }
 
+func TestInspectStringCustomExcludingSubstring(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	var buf bytes.Buffer
+
+	if err := inspectStringCustomExcludingSubstring(&buf, tc.ProjectID, "Name: Doe, John. Name: Example, Jimmy", "[A-Z][a-z]{1,15}, [A-Z][a-z]{1,15}", []string{"Jimmy"}); err != nil {
```

Comment:
Please add an assert to confirm that Jimmy is not detected.

Reply:
- Added assert here : https://github.com/GoogleCloudPlatform/golang-samples/blob/e70120a579e07047f9e94ad8136714563c82d7f0/dlp/snippets/inspect/inspect_test.go#L295